api/app/signals/apps/reporting/csv/utils.py

```diff
@@ -11,7 +11,7 @@
 from django.db import connection
 from django.db.models import Case, CharField, QuerySet, Value, When
 from django.utils import timezone
-from swift.storage import SwiftStorage
+from storages.backends.azure_storage import AzureStorage
```

Comment:
General remark about AzureStorage instead of SwiftStorage. Let's check with the VNG if this change will not break other installations.

Reply:
This is now split up with a feature flag so depending on the usages it will pick either swift, azure or filestorage